pom.xml

```diff
@@ -82,6 +82,24 @@
          </configuration>
        </plugin>
      </plugins>
+    <pluginManagement>
+      <plugins>
+      <plugin>
+        <groupId>org.jenkins-ci.tools</groupId>
+        <artifactId>maven-hpi-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <archive>
+            <index>true</index>
+            <manifestEntries>
+              <Hudson-Version>1.609.18.1</Hudson-Version>
+              <Jenkins-Version>1.609.18.1</Jenkins-Version>
```

Comment:
If you install it on something without the core update the reflection will not find the method and fall through to the old code path, so no worse than today. On the core versions with the method, reflection will find and invoke the method and all will be well.